libraries/azure-app-configuration-importer/src/internal/parsers/defaultConfigurationSettingsConverter.ts

```diff
@@ -259,9 +313,49 @@ class FeatureFlagConfigurationSettingsConverter implements ConfigurationSettings
         `Data contains feature flags in invalid format. The type of ${featureFlagName} should be either boolean or dictionary.`
       );
     }
+
     return featureFlagValue;
   }
 
+  private getFeatureFlagsFromV2Schema(featureFlag: any): FeatureFlagValue {
+    const ajv = new Ajv();
```

Comment:
Seems we don't have to new it every time

Reply:
Good catch, updated